repo/blob/s3/s3_storage_test.go

```diff
@@ -187,8 +197,10 @@ func TestS3StorageMinioSTS(t *testing.T) {
 		t.Skip("endpoint not reachable")
 	}
 
-	createBucket(t, options)
-	testStorage(t, options)
+	testlease.RunWithLease(t, testLeaseID, testLeaseDuration, func() {
```

Comment:
It seems that it is desirable to run this one with a lease. The observation has been that either the created user or generated token disappear under load.